crates/goose/src/providers/bedrock.rs

```diff
@@ -27,13 +27,23 @@ pub const BEDROCK_DEFAULT_MODEL: &str = "anthropic.claude-3-5-sonnet-20240620-v1
 pub const BEDROCK_KNOWN_MODELS: &[&str] = &[
     "anthropic.claude-3-5-sonnet-20240620-v1:0",
     "anthropic.claude-3-5-sonnet-20241022-v2:0",
+    "anthropic.claude-3-7-sonnet-20250219-v1:0",
+    "anthropic.claude-sonnet-4-20250514-v1:0",
+    "anthropic.claude-sonnet-4-20250514-v1:0",
```

Comment:
maybe change the default model too?

Reply:
Does the default Goose install script use latest instead of version pinning? If so, I worry about disruption to existing users by changing the model since their prompts may work worse (or better!) on a new model. Could require tuning prompt(s) and running evals to adopt in some cases, this is really more of an issue for pipeline use cases than desktop CLI/GUI use probably.

What do you think about me logging an Issue for the model update to have that be a separate more visible change?